app-misc/anki/metadata.xml

```diff
@@ -13,4 +13,7 @@
 	<upstream>
```

Comment:
please co-maintain the package too, thanks!

Reply:
Sure!